grpc/tests/src/test/groups.rs

```diff
@@ -12,6 +11,7 @@ use casper_execution_engine::core::{engine_state::Error, execution};
 use casper_types::{
     account::AccountHash, contracts::CONTRACT_INITIAL_VERSION, runtime_args, Key, RuntimeArgs, U512,
 };
+use once_cell::sync::Lazy;
```

Comment:
Should be with other 3rd party crate imports.

Reply:
Moved